pkg/cvo/availableupdates.go

```diff
@@ -187,40 +192,71 @@ func calculateAvailableUpdatesStatus(clusterID string, proxyURL *url.URL, tlsCon
 	currentVersion, err := semver.Parse(version)
 	if err != nil {
 		klog.V(2).Infof("Unable to parse current semantic version %q: %v", version, err)
-		return nil, configv1.ClusterOperatorStatusCondition{
+		return cvoCurrent, nil, configv1.ClusterOperatorStatusCondition{
 			Type: configv1.RetrievedUpdates, Status: configv1.ConditionFalse, Reason: "InvalidCurrentVersion",
 			Message: "The current cluster version is not a valid semantic version and cannot be used to calculate upgrades.",
 		}
 	}
 
-	updates, err := cincinnati.NewClient(uuid, proxyURL, tlsConfig).GetUpdates(upstreamURI, arch, channel, currentVersion)
+	current, updates, err := cincinnati.NewClient(uuid, proxyURL, tlsConfig).GetUpdates(upstreamURI, arch, channel, currentVersion)
 	if err != nil {
 		klog.V(2).Infof("Upstream server %s could not return available updates: %v", upstream, err)
 		if updateError, ok := err.(*cincinnati.Error); ok {
-			return nil, configv1.ClusterOperatorStatusCondition{
+			return cvoCurrent, nil, configv1.ClusterOperatorStatusCondition{
 				Type: configv1.RetrievedUpdates, Status: configv1.ConditionFalse, Reason: updateError.Reason,
 				Message: fmt.Sprintf("Unable to retrieve available updates: %s", updateError.Message),
 			}
 		}
 		// this should never happen
-		return nil, configv1.ClusterOperatorStatusCondition{
+		return cvoCurrent, nil, configv1.ClusterOperatorStatusCondition{
 			Type: configv1.RetrievedUpdates, Status: configv1.ConditionFalse, Reason: "Unknown",
 			Message: fmt.Sprintf("Unable to retrieve available updates: %s", err),
 		}
 	}
 
-	var cvoUpdates []configv1.Update
+	cvoCurrent, err = convertCincinnatiUpdateToRelease(current)
+	if err != nil {
+		return cvoCurrent, nil, configv1.ClusterOperatorStatusCondition{
+			Type: configv1.RetrievedUpdates, Status: configv1.ConditionFalse, Reason: "ResponseInvalid",
+			Message: fmt.Sprintf("Invalid Cinncinnati node: %s", err),
```

Comment:
```suggestion
			Message: fmt.Sprintf("Invalid retrieved update node: %s", err),
```

We should avoid mentioning Cincinnati (:cry:) due to upcoming rename

Reply:
We are renaming our implementation.  The protocol remains "Cincinnati", and the CVO only cares about the protocol, not its backing implementation.  I'd be fine with `recommended update node` though, since that seems more intuitive than mentioning Cincinnati.  I'd rather avoid `retrieved` since that might give a false `image retrieved` impression.  Thoughts?

Reply:
Oh, right, "recommended update" sounds good to me

Reply:
Done as part of 0a0483ae -> cd2d7013.